web-app/src/main/webapp/script/postUserData.js

```diff
@@ -0,0 +1,43 @@
+// This currently gets the address from the report form's location field (no autopopulate, no map picker)
```

Comment:
Technically this is now incorrect, it does autopopulate and there is a map picker